client/src/helpers/stardust/transactionsHelper.ts

```diff
@@ -20,7 +20,11 @@ export interface Input {
 }
 
 export interface Output {
-    index: number; type: 2 | 3 | 4 | 5 | 6; address: IBech32AddressDetails; amount: number; isRemainder: boolean;
+    index: number;
+    type: number;
```

Comment:
Why number? I think is better if we constrict it to possible types.

Reply:
Done.